src/MoBi.UI/Presenters/ReactionDiagramPresenter.cs

```diff
@@ -179,18 +180,16 @@ private bool moleculeNodeWillBeUnlinkedAfterDelete(MoleculeNode moleculeNode, IE
       private void removeItem(GoObject itemToDelete)
       {
          if (itemToDelete.IsAnImplementationOf<MoleculeNode>())
-            RemoveMoleculeNode((MoleculeNode)itemToDelete);
+            RemoveMoleculeNode((MoleculeNode) itemToDelete);
 
          else if (itemToDelete.IsAnImplementationOf<ReactionNode>())
-            RemoveReactionNode((ReactionNode)itemToDelete);
+            RemoveReactionNode((ReactionNode) itemToDelete);
       }
 
       private bool anyLinkedNodes(MoleculeNode moleculeNode)
       {
-         var moleculeNodes = reactionDiagramManager.GetMoleculeNodes(moleculeNode.Name);
          var reactionNodes = getLinkedReactionsForMoleculeNode(moleculeNode);
-
-         return reactionNodes.Any() && moleculeNodes.Count() == 1;
+         return reactionNodes.Any();
```

Comment:
This is the one change. If there were multiple nodes, the prompt was not shown

Reply:
I do not know why it was like this and there are not tests to explain the behavior.  I tried delete and multiple delete, seems to be working fine